packages/core-modules/test/contracts/modules/CommsMixin.test.js

```diff
@@ -1,8 +1,8 @@
-const assert = require('assert/strict');
+const { ethers } = hre;
+const assert = require('assert');
```

Comment:
```suggestion
const assert = require('assert/strict');
```

I like to use the `assert/strict` by default, as it uses the Strict comparisons by default and shows better diffs on error: https://nodejs.org/api/assert.html

Reply:
Moved this to a separate issue: https://github.com/Synthetixio/synthetix-v3/issues/426